src/main/java/org/jitsi/videobridge/Endpoint.java

```diff
@@ -299,6 +302,7 @@ public void trace(@NotNull Function0<Unit> f)
                 }
             });
         bitrateController = new BitrateController(this, diagnosticContext, logger);
+        bitrateController.subscribe(this);
```

Comment:
Can you create an anonymous class to add as the handler?  That way we don't make the API of Endpoint too broad (nothing outside cares or should leverage `Endpoint` as a `BitrateController.BitrateControllerListener`)